### PR TITLE
Add page for socat

### PR DESCRIPTION
**Pull request by dyng**
Hi,

I have created a usage page for [socat](http://www.dest-unreach.org/socat/). Hope this can help someone.

### CODE REVIEW

**Comment by igorshubovych** [comment]
Please check out linter output

**Comment by dyng** [comment]
I have fixed the syntax problem. (so sorry about that)

File: pages/common/socat.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# socat
+
+> Multipurpose relay (SOcket CAT).
+
+- Listen to a port, wait for an incoming connect and transfer data to STDIO:
```
Comment (waldyrious) [inline pages/common/socat.md]: "incoming connection", perhaps?

Comment (dyng) [inline pages/common/socat.md]: Oops.. I have fixed it, should I squash the commits up?

**Comment by waldyrious** [comment]
From my side this is ready to squash and merge. @igorshubovych any further comments?

**Comment by igorshubovych** [comment]
Now, LGTM. Thank you!

**Pull request merged**